Backpack/Font/Classes/BPKFontMapping.m

```diff
@@ -0,0 +1,61 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKFontMapping.h"
+#import <Backpack/Color.h>
+#import <Backpack/Common.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKFontMapping ()
+
+@end
+
+@implementation BPKFontMapping
+
+- (instancetype)initWithFamily:(NSString *)family
+               regularFontFace:(NSString *)regularFontFace
+              semiboldFontFace:(NSString *)semiboldFontFace
+                 heavyFontFace:(NSString *)heavyFontFace {
+    self = [super init];
+    if (self) {
+        _family = [family copy];
+        _regularFontFace = [regularFontFace copy];
+        _semiboldFontFace = [semiboldFontFace copy];
+        _heavyFontFace = [heavyFontFace copy];
+    }
+    return self;
+}
+
+- (NSString *)familyForWeight:(UIFontWeight)weight {
+    if (weight == UIFontWeightRegular) {
```

Comment:
Keep in mind that `UIFontWeight` is a actually a type aliased `CGFloat` so you are comparing floats here. Not so sure about that

Reply:
@k0nserv all done now. Not sure if I changed this the way you expected? https://github.com/Skyscanner/backpack-ios/pull/306/files#diff-f0207de3091521efb61f277e6b8ec8f3R42